draft-ietf-ppm-dap.md

```diff
@@ -505,25 +503,12 @@ parameters used by the Aggregators to select a batch of reports specific to the
 given query type. A query is defined as follows:
 
 ~~~
-enum {
-    next-batch(0),
-    by-batch-id(1),
-} FixedSizeQueryType;
-
-opaque BatchId[32];
-
-struct {
-    FixedSizeQueryType fixed_size_query_type;
-    select (fixed_size_query_type) {
-        next-batch:  Empty;
-        by-batch-id: BatchId batch_id;
-    }
-} FixedSizeQuery;
+uint64 BatchID;
```

Comment:
👍🏻 I restored it to 32 bytes as originally written; I suppose this matches the size of other "opaque identifiers for which we want to enable random generation with negligible collision chance". (I'd advocate for changing all of these identifiers to 16 bytes rather than 32, but I also think that's a separable change, and so for now I think maintaining consistency is more valuable.)

Reply:
I'd be all for 16 bytes, but yes this change is separable :) One thing we need to keep in mind is whether we need IDs to be the hash of things, in which case 32 bytes might be better.